config/initializers/session_store.rb

```diff
@@ -0,0 +1,18 @@
+redis_url = ENV.fetch("REDIS_URL", nil)
+return unless redis_url
```

Comment:
If we return here, what `session_store` will Rails use? I've tested it and it works - but I don't see how!?

Reply:
Okay, looking at the docs it defaults to `:cookie_store`, makes sense now.

Reply:
I was being explicit about this in an earlier version. Whilst I knew it would be the cookie store after dealing with it it's important to note that the next person to read it is likely to wonder the same so I'll add it back in!

Reply:
Done 👍

Reply:
Another advantage of being explicit is that we can set the same session names and expiration dates to provide some consistency.